applications/API/serializers.py

```
@@ -0,0 +1,505 @@
+        if cached_value is not None:
+            return cached_value         
+        
+        if not obj.added_by:
+            return None
+
+        user = obj.added_by
+
+        if user.first_name:
+            if user.last_name:
+                return f"{user.first_name} {user.last_name[0]}."
+            else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)